src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -54,9 +48,9 @@ public PersonCard(ReadOnlyPerson person, int displayedIndex) {
      */
     private void bindListeners(ReadOnlyPerson person) {
         name.textProperty().bind(Bindings.convert(person.nameProperty()));
-        phone.textProperty().bind(Bindings.convert(person.phoneProperty()));
-        address.textProperty().bind(Bindings.convert(person.addressProperty()));
-        email.textProperty().bind(Bindings.convert(person.emailProperty()));
+        //phone.textProperty().bind(Bindings.convert(person.phoneProperty()));
+        //address.textProperty().bind(Bindings.convert(person.addressProperty()));
+        //email.textProperty().bind(Bindings.convert(person.emailProperty()));
```

Comment:
Removing the lines is fine, no need to comment. We can refer to revision history if we need it back.